### PR TITLE
Fix docs for 'complete' field in 'signrawtransaction' response

### PR DESCRIPTION
**Pull request by charlescharles**

### CODE REVIEW

File: src/rpcrawtransaction.cpp [hunk 1]
```diff
@@ -530,7 +530,8 @@ Value signrawtransaction(const Array& params, bool fHelp)
             "\nResult:\n"
             "{\n"
             "  \"hex\": \"value\",   (string) The raw transaction with signature(s) (hex-encoded string)\n"
-            "  \"complete\": n       (numeric) if transaction has a complete set of signature (0 if not)\n"
+            "  \"complete\":
+                    true|false       (boolean) if transaction has a complete set of signature\n"
```
Comment (jonasschnelli) [inline src/rpcrawtransaction.cpp]: This won't compile. Did you try compiling this before creating the PR?

**Comment by laanwj** [comment]
Concept ACK, but you'll need to make sure this compiles and works too.

**Comment by charlescharles** [comment]
Fixed, it compiles for me now. Sorry about that; it seemed like a trivial change and I was careless.

**Comment by laanwj** [comment]
ACK, can go into trival branch @theuni

**Comment by fanquake** [comment]
ACK

**Comment by theuni** [comment]
Merged to trivial-next, this can be closed.

**Comment by charlescharles** [comment]
thanks!

**Pull request closed without merging**